settings.py

```diff
@@ -33,9 +33,13 @@
 
 HOMEPAGE = "http://chaosthebot.com"
 
-# TEST SETTING PLEASE IGNORE
+# TEST SETTING
```

Comment:
The author of this forgot to say add `TEST COMMENT PLEASE IGNORE`. Classic mistake.